lib/fastlane/actions/carthage.rb

```diff
@@ -68,20 +82,29 @@ def self.available_options
                                        verify_block: proc do |value|
                                          raise "Please pass a valid value for no_build. Use one of the following: true, false" unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
                                        end),
-          FastlaneCore::ConfigItem.new(key: :verbose,
-                                       env_name: "FL_CARTHAGE_VERBOSE",
-                                       description: "Print xcodebuild output inline",
+          FastlaneCore::ConfigItem.new(key: :use_ssh,
+                                       env_name: "FL_CARTHAGE_USE_SSH",
+                                       description: "Use SSH for downloading GitHub repositories",
                                        is_string: false,
                                        optional: true,
                                        verify_block: proc do |value|
-                                         raise "Please pass a valid value for verbose. Use one of the following: true, false" unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
+                                         raise "Please pass a valid value for use_ssh. Use one of the following: true, false" unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
                                        end),
-          FastlaneCore::ConfigItem.new(key: :platform,
-                                       env_name: "FL_CARTHAGE_PLATFORM",
-                                       description: "Define which platform to build for",
+          FastlaneCore::ConfigItem.new(key: :use_submodules,
+                                       env_name: "FL_CARTHAGE_USE_SUBMODULES",
+                                       description: "Add dependencies as Git submodules",
+                                       is_string: false,
                                        optional: true,
                                        verify_block: proc do |value|
-                                         raise "Please pass a valid platform. Use one of the following: all, iOS, Mac, watchOS" unless ["all", "iOS", "Mac", "watchOS"].include? value
+                                         raise "Please pass a valid value for use_submodules. Use one of the following: true, false" unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :no_use_binaries,
+                                       env_name: "FL_CARTHAGE_NO_USE_BINARIES",
+                                       description: "Check out dependency repositories even when prebuilt frameworks exist",
+                                       is_string: false,
+                                       optional: true,
+                                       verify_block: proc do |value|
```

Comment:
I did in this PR:
1. add `command` option to support `carthage build`
2. add options `configuration`, `no_skip_current` and `no_checkout`.
3. `use_binaries` to `no_use_binaries` to follow the `carthage` itself options.
4. Sorting `cmd <<`, `FastlaneCore::ConfigItem.new`, and rspec as the same order.

The most part is 4.